src/main/java/org/example/onlinebootstore/model/Book.java

```diff
@@ -0,0 +1,30 @@
+package org.example.onlinebootstore.model;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+import java.math.BigDecimal;
+
+@Setter
+@Getter
+@Entity
+@Table(name = "book")

```

Comment:
table name should be plural according to the naming convention
```suggestion
@Table(name = "books")
```